config/environments/production_aks.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require Rails.root.join('config/environments/production')
```

Comment:
Can you clarify why we need another rails env, please?

Reply:
I think to keep things separate distinct between the different environments. It's the same pattern we have followed for Apply and Register.